packages/workspace/src/tasks-runner/run-command/index.ts

```diff
@@ -104,6 +115,16 @@ export async function runCommand<T extends RunArgs>(
   }
   const lifeCycle = new CompositeLifeCycle(lifeCycles);
 
+  let inkWaitUntilExit: () => Promise<void> | undefined;
+  if (process.env.NX_TASKS_RUNNER_USE_INK === 'true') {
+    const { waitUntilExit } = render(
+      createElement(RunCommandComponent, {
+        lifeCycle,
```

Comment:
Since the component only cares about the ink lifecycle object, I don't like passing the composite one inside and then searching for the ink one inside. We are doing wrapping and unwrapping, even though we know if the ink one is used.

I'd rather assign the terminal output lifecycle to a var:

output = getTerminalOutputLifeCycle(
initiatingProject,
terminalOutputStrategy,
projectsToRun,
tasks,
nxArgs,
overrides
),

and then pass it to createElement as a prop, and then just assume inside it's the ink one. Cause we are using hte same if statement to create the lifecycle and to decide whether to render the component.